plugins/woocommerce-admin/client/products/fields/variations/variations.scss

```diff
@@ -1,5 +1,17 @@
 .woocommerce-product-variations {
-	min-height: 300px;
+	ol {
+		@media ( min-width: #{ ($break-medium) } ) {
+			&.rows-5 {
+				min-height: 420px;
+			}
+			&.rows-10 {
+				min-height: 840px;
+			}
+			&.rows-25 {
+				min-height: 2100px;
```

Comment:
This and the 10 row height both feel a bit odd to me.  The card extends beyond the viewport of the page and has a lot of whitespace.

Maybe we could just include the min-height of 420px.  Any thoughts @jarekmorawski?
<img width="596" alt="Screen Shot 2022-12-16 at 8 56 21 AM" src="https://user-images.githubusercontent.com/10561050/208150190-21ef666a-e591-4062-99e6-87bb9dbf499b.png">

Reply:
Yes, that's a good idea. 420 px sounds good! To be clear, it would only apply when the user changes the number of items on a page but there aren't enough of them to fill the table, correct?

Reply:
This would set a minimum height (to larger screens) of 420px when not enough items are there to fill.

I think we can forego the larger heights if that's okay with you since that probably implies they have enough items to fill the list anyway and would create the weird blank space shown above.

Reply:
Got it! I added a minimum height of 420px for larger screens, no matter the items that will be shown in the list.
@joshuatf could you take another look at this PR?